flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/TimeFunctionsITCase.java

```diff
@@ -947,6 +947,14 @@ private Stream<TestSetSpec> toTimestampLtzTestCases() {
                                         .atZone(ZoneId.of("America/Los_Angeles"))
                                         .toInstant(),
                                 TIMESTAMP_LTZ(3).nullable())
```

Comment:
Address this comment: 
https://github.com/apache/flink/pull/25763#discussion_r1892889885

Reply:
Makes sense to me.

Reply:
lgtm.